config.yaml.example

```diff
@@ -2,8 +2,8 @@
 # "none", "Baaj", "Besaid", "Boat1", "Kilika", "Luca", "Miihen", "MRR"
 # "Djose", "Moonflow", "Guadosalam", "Macalania", "Home", "rescue_yuna"
 # "Gagazet", "Zanarkand", "Sin", "Nem_Farm"
-gamestate               : "none" # valid values listed above
-step_counter            : 1      # valid step_counter values are dependent on gamestate
+gamestate               : "Nem_Farm"  # valid values listed above
```

Comment:
These should probably be `none` and `. Rather than throwing new folks into a nemesis run. Lol